docs/guides/conda.rst

```diff
@@ -8,16 +8,171 @@ and need them installed when building their documentation.
 
 This work was funded by `Clinical Graphics`_ -- many thanks for their support of Open Source.
 
+.. _Clinical Graphics: https://www.clinicalgraphics.com/
+
 Activating Conda
 ----------------
 
 Conda support is available using a :doc:`../config-file/index`, see :ref:`config-file/v2:conda`.
 
-This Conda environment will also have Sphinx and other build time dependencies installed.
-It will use the same order of operations that we support currently:
+Our :ref:`Docker images <builds:Docker images>` use Miniconda, a minimal conda installer.
+After specifying your project requirements using conda ``environment.yml`` files,
+Read the Docs will create the environment (using ``conda env create``)
+and add the core dependencies needed to build the documentation.
+
+Creating the ``environment.yml``
+--------------------------------
+
+There are several ways of `exporting a conda environment`_:
+
+- ``conda env export`` will produce a complete list of all the packages installed in the environment
+  with their exact versions. This is the best option to ensure reproducibility,
+  but can create problems if done from a different operative system than the target machine.
+- ``conda env export --from-history`` will only include packages that were explicitly requested,
+  excluding the transitive dependencies. This is the best option to maximize cross-platform compatibility.
+- And finally, you can also write it by hand. This allows for more fine-tuned control and overcomes
+  some limitations in the conda exporting capabilities.
+
+For example, using the second method for an existing environment:
+
+.. code-block::
+
+    $ conda activate rtd38
+    (rtd38) $ conda env export --from-history | tee environment.yml
```

Comment:
is there any problem with using `>`?

```suggestion
    (rtd38) $ conda env export --from-history > environment.yml
```

Reply:
I wanted to both show it on screen and save it to `environment.yml`, hence the use of `tee`